db/seeds.rb

```diff
@@ -65,44 +65,98 @@ def puts_records_in_db(klass)
+
+illinois = State.find_or_create_by!(name: 'Illinois', abbr: 'IL')
+cook_county = County.find_or_create_by!(name: 'Cook', state: illinois)
+cook_zip_60686 = Zipcode.find_or_create_by!(state: cook_county.state, county: cook_county, code: '60686', city: 'Chicago')
+cook_zip_60688 = Zipcode.find_or_create_by!(state: cook_county.state, county: cook_county, code: '60688', city: 'Chicago')
+
+lake_county = County.find_or_create_by!(name: 'Lake', state: illinois)
 
 # ---------------------------------------------
 # Businesses
 # ---------------------------------------------
 
-@business = Business.where(name: 'Happy Seedlings Childcare', user: @user_kate).first_or_create(
+@biz_happy_seedlings = Business.where(name: 'Happy Seedlings Childcare', user: @user_kate).first_or_create(
   license_type: Licenses.types.keys.first,
   county: big_horn_cty_mt,
   zipcode: hardin_zipcode
 )
 
+@biz_goslings = Business.where(name: 'Goslings Grow', user: @user_kate).first_or_create(
+  license_type: Licenses.types.keys.first,
+  county: cook_county,
+  zipcode: cook_zip_60688
+)
+@biz_lil_ducks = Business.where(name: 'Lil Baby Ducklings', user: @user_kate).first_or_create(
+  license_type: Licenses.types.keys.first,
+  county: cook_county,
+  zipcode: cook_zip_60686
+)
+
 puts_records_in_db(Business)
 
+# ---------------------------------------------
+# Approvals
+# ---------------------------------------------
+
+Approval.create!(case_number: '1234567',
+                 copay: 500.00,
+                 copay_frequency: 'weekly',
+                 effective_on: Date.new(2019,11,12),
+                 expires_on: Date.new(2020,11,11)
+                 )
+Approval.create!(case_number: '1231231',
+                 copay: 1111.11,
+                 copay_frequency: 'monthly',
+                 effective_on: Date.new(2020,2,4),
+                 expires_on: Date.new(2021,2,3)
+)
+Approval.create!(case_number: '4567890',
+                 copay: 1500.00,
+                 copay_frequency: 'monthly',
+                 effective_on: Date.new(2020,2,4),
+                 expires_on: Date.new(2021,2,3)
+)
+Approval.create!(case_number: '4242424',
+                 copay: 555.55,
+                 copay_frequency: 'weekly',
+                 effective_on: Date.new(2020,1,4),
+                 expires_on: Date.new(2021,1,3)
+)
+puts_records_in_db(Approval)
+
 # ---------------------------------------------
 # Children w/ Required Approvals
 # ---------------------------------------------
 
 # find_or_create_by! a Child with the full_name,
 #  and birthday set randomly between the min_age and max_age.
-def child_named(full_name, min_birthday: MIN_BIRTHDAY,
-                max_birthday: MAX_BIRTHDAY,
-                business: @business)
-  effective_date = Faker::Date.between(from: 1.year.ago, to: Time.zone.today)
+def child_named(full_name, date_of_birth: nil,
```

Comment:
How come we're defaulting to nil here rather than the Faker.between?  Kids are going to be required to have dob on entry.